relx.config

```diff
@@ -8,7 +8,6 @@
   {exml, load},
```

Comment:
Why do we get rid of these dependencies?

Reply:
Release will not be build otherwise. It seems that deps of deps were added into `relx.config`